service_management/test/controllers/analysis/SampleObjectControllerIntegrationSpec.scala

```diff
@@ -188,6 +189,13 @@ class SampleObjectControllerIntegrationSpec
       validateSampleObject(ujs, res.json)
     }
 
+    "list all treatments" in {
```

Comment:
That also means making separate `Treatment*Spec` implementations for the different test cases